js/main.js

```diff
@@ -27,6 +28,162 @@ Site = {
     });
   },
 
+  hideLoading: function() {
+    $('#loading-screen').addClass('hide');
+  }
+
+
+};
+
+Site.Chairs = {
+  tail: [],
+  tailSize: 6,
+  // listSize: 86400, // = 24hrs
+   listSize: 120, // USE THIS FOR DEV
+  bufferTime: 3000, // in ms
+  imgDirPath: '/chairs/',
+  init: function() {
+    var _this = this;
+
+    _this.imagesContainer = document.querySelector('#images-container');
+
+    // _this.currentPosition = _this.getSecsToday + _this.bufferTime;
+    _this.currentPosition = 3; // USE THIS FOR DEV
+
+    _this.getList();
+    _this.initTail();
+  },
+
+  getList: function() {
+    var _this = this;
+
+    _this.list = _this.generateRandomList();
+  },
+
+  generateRandomList: function() {
+    var _this = this;
+
+    var list = [];
+
+    for (var i = 0; i < _this.listSize; i++) {
+      list[i] = i + 1;
+    }
+
+    var top = list.length;
+    if(top) {
+      while(--top) {
+        var current = Math.floor(Math.random() * (top + 1));
+        var tmp = list[current];
+        list[current] = list[top];
+        list[top] = tmp;
+      }
+    }
+
+    return list;
+  },
+
+  initTail: function() {
+    var _this = this;
+
+    _this.loadTail();
+    _this.loadImages();
+    _this.triggerTimer();
+  },
+
+  loadTail: function() {
+    var _this = this;
+
+    for(var i = 0; i <= _this.tailSize; i++) {
+      _this.tail[i] = _this.list[i + _this.currentPosition];
+    }
+  },
+
+  loadImages: function() {
+    var _this = this;
+
+    for(var image of _this.tail) {
```

Comment:
good question. no. i rather rewrite this part than adding transpiling to the project.